### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [imattdu/imattdu.github.io#2] [Announcements]
# index

matt note

http://127.0.0.1:8000/

**Comment by imattdu** [comment]
es

https://blog.csdn.net/lilyssh/article/details/119646563
https://blog.csdn.net/sinat_36053757/article/details/117679600

lsof -nP -p 27289 | grep LISTEN

brew install tcpdump


brew services start elastic/tap/elasticsearch-full

brew services start elastic/tap/kibana-full

tcpdump host 127.0.0.1 and port 90

**Reply by imattdu** [reply]
111

**Reply by imattdu** [reply]
1

**Comment by imattdu** [comment]
vo,bo,dto
https://zhuanlan.zhihu.com/p/102389552
json2go
https://oktools.net/json2go

**Comment by imattdu** [comment]
https://www.jianshu.com/p/75e90aa8f07e
如果没有传map字段则为nil

**Comment by imattdu** [comment]
https://github.com/alibaba/p3c/blob/master/Java%E5%BC%80%E5%8F%91%E6%89%8B%E5%86%8C(%E9%BB%84%E5%B1%B1%E7%89%88).pdf


mac:top 0 mem cpu
linux top m p


https://blog.csdn.net/qq_33858250/article/details/81047883

**Comment by imattdu** [comment]
git clone git@git.n.xxx.com:xxx/xxx.git

git fetch origin dev    命令来把远程dev分支拉到本地

git checkout -b dev origin/dev   在本地创建分支dev并切换到该分支

git pull origin dev 就可以把远程dev分支上的内容都拉取到本地了



git fetch origin master
git merge origin/master


git branch -d xx
删除远程分支
git push origin --delete xx

git log
git log --oneline


https://zhuanlan.zhihu.com/p/433213071


https://books.studygolang.com/The-Golang-Standard-Library-by-Example/chapter03/03.1.html#311-%E6%95%B0%E6%8D%AE%E9%9B%86%E5%90%88%E6%8E%92%E5%BA%8F

**Comment by imattdu** [comment]
https://github.com/go-delve/delve/tree/master/Documentation/installation

>=1.10
go build -gcflags "all=-N -l" ./main.go
<=1.9
go build -gcflags "-N -l" github.com/app/demo


run
dlv --check-go-version=false --listen=:2345 --headless=true --api-version=2 --accept-multiclient exec ./main --config ./cfg/xxx.cfg



git clone git@git.n.xxx.com:xxx/xxx.git

git fetch origin dev 命令来把远程dev分支拉到本地

git checkout -b dev origin/dev 在本地创建分支dev并切换到该分支

git pull origin dev 就可以把远程dev分支上的内容都拉取到本地了

git fetch origin master
git merge origin/master

git branch -d xx
删除远程分支
git push origin --delete xx

git log
git log --oneline


https://blog.csdn.net/kld230/article/details/106917686/

**Comment by imattdu** [comment]
https://medium.com/wesionary-team/understanding-casbin-with-different-access-control-model-configurations-faebc60f6da5


https://blog.csdn.net/huanby/article/details/123103191



git rm -r --cached .idea

https://cloud.tencent.com/developer/article/1574114


text 不可以为not null, default


https://blog.csdn.net/weixin_34159110/article/details/86262368

**Comment by imattdu** [comment]
vim:
https://zhuanlan.zhihu.com/p/89042423




:s/old/new      //用new替换行中首次出现的old
:s/old/new/g         //用new替换行中所有的old
:n,m s/old/new/g     //用new替换从n到m行里所有的old
:%s/old/new/g      //用new替换当前文件里所有的old


/keyword     //向光标下搜索keyword字符串，keyword可以是正则表达式
?keyword     //向光标上搜索keyword字符串
n           //向下搜索前一个搜素动作
N         //向上搜索前一个搜索动作


u        //撤销上一步操作
U        //撤销对当前行的所有操作

**Comment by imattdu** [comment]
跨域
https://blog.csdn.net/weixin_45943355/article/details/125223919
https://blog.csdn.net/qq_40739917/article/details/123862470

**Comment by imattdu** [comment]
跨域
https://blog.csdn.net/weixin_45943355/article/details/125223919
https://blog.csdn.net/qq_40739917/article/details/123862470


https://blog.wolfogre.com/posts/go-ppof-practice/#%E6%8E%92%E6%9F%A5%E9%94%81%E7%9A%84%E4%BA%89%E7%94%A8

**Comment by imattdu** [comment]
master
develop
feature-user
feature-user-fix
hotfix-user master
bugfix-user develop

feat：新功能（feature）
fix：修补bug
docs：文档（documentation）
style： 格式（不影响代码运行的变动）
refactor：重构（即不是新增功能，也不是修改bug的代码变动）
test：增加测试
chore：构建过程或辅助工具的变动

https://zhuanlan.zhihu.com/p/182553920

top
P M
o cpu rsize

**Comment by imattdu** [comment]
https://blog.csdn.net/peanutwzk/article/details/93714646

**Comment by imattdu** [comment]
jwt:
https://github.com/golang-jwt/jwt

https://developer.aliyun.com/article/363081

**Comment by imattdu** [comment]
https://zhuanlan.zhihu.com/p/334720992

https://segmentfault.com/a/1190000022321518

**Comment by imattdu** [comment]
selenium

**Comment by imattdu** [comment]
https://segmentfault.com/a/1190000007748862

**Comment by imattdu** [comment]
yum groupinstall Fonts

**Comment by imattdu** [comment]
FROM         centos:centos7

RUN yum install -y epel-release
RUN yum install -y gcc gcc-c++ kernel-devel make
RUN yum -y groupinstall "Development tools"
RUN yum -y install wget zlib-devel bzip2-devel openssl-devel ncurses-devel sqlite-devel readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel
RUN wget https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tar.xz
RUN mkdir /usr/local/python3

RUN tar -xvJf  Python-3.8.0.tar.xz \
&& cd Python-3.8.0 \
&& ./configure --prefix=/usr/local/python3 \
&& make && make install \
&& ln -s /usr/local/python3/bin/python3 /usr/bin/python3 \
&& ln -s /usr/local/python3/bin/pip3 /usr/bin/pip3

**Comment by imattdu** [comment]
https://docs.nebula-graph.com.cn/3.2.1/2.quick-start/1.quick-start-workflow/

https://www.abbreviations.com/abbreviation/string

**Comment by imattdu** [comment]
https://zhuanlan.zhihu.com/p/102389552

**Comment by imattdu** [comment]
https://blog.csdn.net/ljj654/article/details/113878588?spm=1001.2101.3001.6650.2&utm_medium=distribute.pc_relevant.none-task-blog-2%7Edefault%7ECTRLIST%7ERate-2-113878588-blog-104820076.pc_relevant_recovery_v2&depth_1-utm_source=distribute.pc_relevant.none-task-blog-2%7Edefault%7ECTRLIST%7ERate-2-113878588-blog-104820076.pc_relevant_recovery_v2&utm_relevant_index=3

**Comment by imattdu** [comment]
https://segmentfault.com/a/1190000022110134

**Comment by imattdu** [comment]
./clickhouse-server --config-file=/etc/clickhouse-server/config.xml


root用户启动

**Comment by imattdu** [comment]
1111
